apps/mull-api/src/app/event/event.resolver.ts

```diff
@@ -31,4 +31,9 @@ export class EventResolver {
   async deleteEvent(@Args('id', { type: /* istanbul ignore next */ () => Int }) id: number) {
     return this.eventService.delete(id);
   }
+
+  @Mutation(() => Event)
+  async addParticipantToEvent(@Args('eventId') eventId: number, @Args('userId') userId: number) {
+    return this.eventService.addParticipant(eventId, userId);
```

Comment:
I've been getting more familiar with GraphQL and I think it would be better to just return a boolean so that we don't have to add fields in the mutation.